StatisticsInR/2.1_simple_linear_regression.Rmd

```diff
@@ -0,0 +1,1223 @@
+
+</br>
+
+## <i class="far fa-question-circle"></i> Which questions? 
+
+</br>
+
+Example questions you can answer with linear regression:
+
+Inference
+ 
+ * How does mean annual temperature change with time?
+ * How does the time of breeding for a bird change with mean spring temperature?
+ * How does relative plant biomass change with mean light intensity?
+
+ 
+Prediction
+
+ * What will the mean summer temperature be in 2100?
+ * How heavy will a sheep be if it is 100cm long?
+
+ 
+</br>
+
+## <i class="fas fa-table"></i> Type of data {.tabset .tabset-fade}
+
+</br>
+
+### Theory
+
+</br>
+
+A linear regression is used when you have continuous numeric response variable and a continuous numeric explanatory variables. A **simple linear regression** has only one explanatory variable, a **multiple linear regression** has more than one. 
+</br>
+
+**Examples of continuous numeric variables**:
+
+* Mean annual temperature
+* Total annual precipitation
+* Distance
+* Height
+* Weight
+
+</br>
+
+**Always remember to check that the way your variables are classified in R is the same as the
+format you expect.** It is a common mistake for a variable that should be numeric
+to be classified as a Factor, or a Factor as a character string etc.  
+
+</br>
+
+### Worked example
+
+</br>
+
+For this worked example we will be using some data on dive depths of
+marine mammals. We will be answering the question:
+
+**Does body size (kg) influence maximum dive depth (m) in marine mammals?**
+
+We might expect that differences in foraging strategies, physiological processes, metabolism,
+and power of the marine mammals would impact how deep the animals can dive. All of these variables correlate with body
+size. Therefore, it could be expected that larger species can dive deeper as relative
+movement compared to body size is lower. Or perhaps it is less energetically demanding
+for smaller species to dive deeper. The direction and strength of the influence of body size on dive depth can be quantified using a linear regression. 
+
+<img src="./Figures/MarineMammals.png" height="150">
+*Figure 1: Illustration of marine mammals by Emily G. Simmonds.*
+
+</br>
+
+#### Introduction to the data
+
+</br>
+
+These data have four variables; species, maximum recorded dive depth in metres, 
+body weight in kilograms, and taxonomic group. 
+
+The data were collected from a variety of sources and cover several taxonomic 
+groups (including
+polar bear, sea otters, baleen whales, toothed whales, seals, and sea lions). You can find the dataset [here]() if you want to try the analysis for yourself. 
+
+</br>
+
+<details><summary> Full list of data sources</summary>
+
+Sources: 
+
```

Comment:
It is good to show all these, but if this is a dataset that we will use several times, we could have an extra page to explain the data and show all the refs. It would shorten this page a bit.
We can discuss this in the group if it makes sense to have a separate page for all the datasets that we use.

Reply:
Let's discuss this. I can also add the references properly as citations.

Reply:
Oki, also discussion point for the next meeting! I'll start to make a list.

Reply:
Thank you!